src/plugins/simple-pricing/xforms/xformProductVariantPricing.js

```diff
@@ -0,0 +1,26 @@
+import accounting from "accounting-js";
+import getDisplayPrice from "../util/getDisplayPrice.js";
+
+export default (node) => {
+  let pricing = {};
+  const { price } = node;
+  if (typeof price === "object") {
+    pricing = {
+      compareAtPrice: null,
+      displayPrice: getDisplayPrice(price.min, price.max),
```

Comment:
I think you need to pass in some currency info as the 3rd param to get currency symbols in the string.